code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -31,6 +31,54 @@
 	new/obj/structure/fluff/empty_terrarium(get_turf(src))
 	return ..()
 
+/obj/effect/mob_spawn/human/dwarven_sarcophagus
+	name = "ancient sarcophagus"
+	desc = "An ancient artifact of greater past. Emits fiery energy"
+	mob_name = "a dwarf"
+	icon = 'icons/obj/dwarven.dmi'
+	icon_state = "sarcophagus"
+	density = TRUE
+	roundstart = FALSE
+	death = FALSE
+	mob_species = /datum/species/human
+	outfit = /datum/outfit/dwarven
+	short_desc = "You are an ancient dwarf awakened from your slumber to regrow your once grand empire."
+	flavour_text = "Milennia have passed since your grand civilization last stood. \
+	You remember the prophecies of a great disaster, in fear of death you volountereed for you blood to be sealed within the lava sarcophagus. \
+	Last thing you remember is a bright light before you awakened from slumber, even tough the sarcophagus is damaged it can still revive your dead brotheren. \
+	Now you are filled with the desire to expand and regrow your civilization once more. Expand, sell artifacts and hunt local fauna while becoming new kings of this ashen land. Strike the earth!"
+	assignedrole = "Dwarf"
+
+/obj/effect/mob_spawn/human/dwarven_sarcophagus/Initialize(mapload)
+	. = ..()
+	var/area/A = get_area(src)
+	notify_ghosts("A Dwarf is ready to awaken at [A.name]", source = src, action=NOTIFY_ATTACK, flashwindow = FALSE, ignore_key = POLL_IGNORE_DWARF)
+
+
+/obj/effect/mob_spawn/human/dwarven_sarcophagus/special(mob/living/new_spawn)
+	var/dwarven_name = pick("Ognog", "Lorenzo", "Bakarat", "Cercer", "Aluminium", "Iro", "Ido", "Kochko", "Bahr", "Mozz", "Fercer", "Bat", "Rot", "Lavaan", "Gorg", "Philzer", "Lerh", \
+	"Vinth", "Gharo","Logreg", "Stawgar", "Lirko", "Ontksu", "Crakta", "Larka", "Dorgo") + " " +pick("Iron","Mithril","Strong","The ","Root","Void","Blood","Titanium","Uranium", \
```

Comment:
called mythril, not mithril in ss13, also a lot of the names you can think of sound really bad

Reply:
i really know i made em in a rush

Reply:
Just do DF dwarf names

Reply:
ill leave it as it is